chain/client/src/info.rs

```diff
@@ -100,6 +101,7 @@ impl InfoHelper {
         validator_epoch_stats: Vec<ValidatorProductionStats>,
         epoch_height: EpochHeight,
         protocol_upgrade_block_height: BlockHeight,
+        statistics: Option<StoreStatistics>,
```

Comment:
Nice collateral benefit of strong types with `statistics: Option<String>` it wasn’t clear what the stats are, and now it is!